.github/CODEOWNERS

```diff
@@ -34,3 +34,6 @@
 
 # Contracts
 /packages/contracts-bedrock     @ethereum-optimism/contract-reviewers
+
+# Security docs
+/docs @ethereum-optimism/evm-safety
```

Comment:
I changed this, but why does this give an error? the team is here https://github.com/orgs/ethereum-optimism/teams/evm-safety